x-pack/plugins/security_solution/server/lib/detection_engine/rule_response_actions/osquery_response_action.ts

```diff
@@ -27,7 +27,7 @@ export const osqueryResponseAction = (
   const { savedQueryId, packId, queries, ecsMapping, ...rest } = responseAction.params;
 
   if (!containsDynamicQueries) {
-    return osqueryCreateAction({
+    return osqueryCreateActionService?.create({
```

Comment:
Looks like `getScheduleNotificationResponseActionsService` should be always available here with only `create` in it, even though the type says it may be `undefined`.

Reply:
👍